tests/rkt_tests.go

```diff
@@ -312,6 +312,31 @@ func runRktAsUidGidAndCheckOutput(t *testing.T, rktCmd, expectedLine string, exp
 	}
 }
 
+func runRkt(t *testing.T, rktCmd string, uid, gid int) (string, int) {
+	child, err := gexpect.Command(rktCmd)
+	if err != nil {
+		t.Fatalf("cannot exec rkt: %v", err)
+	}
+	if gid != 0 {
```

Comment:
+1

On Thu, May 26, 2016 at 1:01 PM, Sergiusz Urbaniak <notifications@github.com
> wrote:

> In tests/rkt_tests.go
> <https://github.com/coreos/rkt/pull/2656#discussion_r64727361>:
>
> > @@ -312,6 +312,31 @@ func runRktAsUidGidAndCheckOutput(t *testing.T, rktCmd, expectedLine string, exp
> >  	}
> >  }
> >
> > +func runRkt(t *testing.T, rktCmd string, uid, gid int) (string, int) {
> > +	child, err := gexpect.Command(rktCmd)
> > +	if err != nil {
> > +		t.Fatalf("cannot exec rkt: %v", err)
> > +	}
> > +	if gid != 0 {
>
> Ok, since this is prior art I suggest we investigate this in another issue.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/2656/files/9732ab555cacc014821839f0938cfcf6988da3e5#r64727361>
>